crates/figma_import/tests/test_fetches.rs

```diff
@@ -83,3 +83,9 @@ fn fetch_dials_gauges() {
         ],
     );
 }
+
+#[test]
+#[cfg_attr(not(feature = "test_fetches"), ignore)]
+fn fetch_styled_text_runs() {
+    run_test("mIYV4YsYYaMTsBMCVskA4N", &["#MainFrame"])
```

Comment:
what does this test that isn't tested already?

Reply:
I had what turned out to be ephemeral fetching issues when I was trying to update the docs, so I had added the test and I think it's worth keeping.

Really I think we should have Rust fetch tests for most to all of our validation docs, but syncing the node names will be an issue so I haven't tried to set up anything like that.